datastore/gcloud/aio/datastore/value.py

```diff
@@ -2,11 +2,15 @@
 from typing import Any
 from typing import Dict
 
+from gcloud.aio.datastore.constants import TypeName
 from gcloud.aio.datastore.constants import TYPES
+from gcloud.aio.datastore.key import Key
 
 
 # https://cloud.google.com/datastore/docs/reference/data/rest/v1/projects/runQuery#value
 class Value:
+    key_kind = Key
```

Comment:
We could add `enitty_kind` here.
There is a circular dependency between Value and Entity so I decided to deal with that when I have to and not before :)